bot/kodiak/evaluation.py

```diff
@@ -96,27 +136,57 @@ def get_merge_body(config: V1, pull_request: PullRequest) -> MergeBody:
             merge_body.commit_message = pull_request.url
         else:
             merge_body.commit_message += "\n\n" + pull_request.url
+
+    co_author_trailers = []
```

Comment:
nit: in keeping with the other usages I think this would be better called `coauthor_trailers`